.gitlab/benchmarks/macrobenchmarks.yml

```diff
@@ -394,6 +394,177 @@ profiler_cpu_timer_create-arm64:
   variables:
     NATIVE_PROFILER_PATH: "dd-trace-dotnet/tracer/tracer-home-linux-arm64/linux-arm64/Datadog.Trace.ClrProfiler.Native.so"
     TRACER_HOME_PATH: "dd-trace-dotnet/tracer/tracer-home-linux-arm64"
+    COR_ENABLE_PROFILING: 1
+    CORECLR_ENABLE_PROFILING: 1
+    DD_PROFILING_ENABLED: 1
+    DD_PROFILING_WALLTIME_ENABLED: 0
+    DD_PROFILING_CPU_ENABLED: 1
+    COMPlus_EnableDiagnostics: 1
+    ENDPOINT: "hello"
+
+.benchmarks-win:
+  stage: benchmarks-win
+  needs: ["check_azure_pipeline"]
+  tags: ["runner:docker"]
+  interruptible: true
+  timeout: 1h
+  # Image based on earlier version of tooling
+  image: registry.ddbuild.io/images/benchmarking-platform-tools-ubuntu:dotnet-tests
+
+  script:
+    - export DD_API_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.$CI_PROJECT_NAME.dd_api_key --with-decryption --query "Parameter.Value" --out text)
```

Comment:
Hey, do you think we can encapsulate it a lil bit better? I would prefer to have a single script call, similar to what we have in https://github.com/DataDog/dd-trace-dotnet/blob/master/.gitlab/benchmarks/macrobenchmarks.yml#L49, SSM details should really belong inside benchmarking-platform repo (and in long term, we should move it to bp-runner, as this seems to be a code used universally for Windows runners).

Reply:
Everything else looks good, we have results in DD dashboard! :tada: 
We will need to do some audit of results, it is 4x faster than Linux? Seems interesting!

Reply:
> it is 4x faster than Linux? Seems interesting

Are these running on the same spec machines?! Because that seems very suspect to me😅 For comparison, [our previous crank-based implementation](https://ddstaging.datadoghq.com/dashboard/c5g-i7d-kcy/ciapp-apm-net-throughput-tests?fromUser=true&refresh_mode=sliding&tpl_var_app_name=%2A&tpl_var_branch%5B0%5D=%28master%20OR%20benchmarks%5C%2F%2A%29&view=spans&from_ts=1734793961810&to_ts=1742569961810&live=true) showed that Windows was ~10% slower than Linux

Reply:
Yep, same spec, m5.metal. Thank you for confirming Andrew, we definitely need to look into it!

 Also, thanks for sharing, that is really lovely dashboard! :raised_hands: :purple_heart:

Reply:
Interesting, is cross-os comparison one of the use-cases? 

> Are these running on the same spec machines?!

Same spec machines yes, although on windows we assign a couple more cpus to the throughput server (anticipating windows being less efficient) . 
Now that I think about it, I will revert back to having the same setup across os to ensure we can have the option if wanted!

Reply:
Here - https://ddstaging.datadoghq.com/dashboard/rgu-p7u-5fq?fromUser=false&refresh_mode=sliding&tpl_var_bp_branch%5B0%5D=fayssal%2Fmigrate-throughput-tests-windows&from_ts=1739975186532&to_ts=1742567186532&live=true

I think we should address it before giving green light to a PR.

Reply:
> although on windows we assign a couple more cpus to the throughput server (anticipating windows being less efficient) .

That must be it.
But it won't hurt to check if instrumentation is applied. Overhead numbers are super small.